universe/Pathfinder.cpp

```diff
@@ -1010,6 +1015,51 @@ std::multimap<double, int> Pathfinder::PathfinderImpl::ImmediateNeighbors(int sy
 }
 
 
+void Pathfinder::PathfinderImpl::WithinJumpsCacheHit(
+    std::unordered_set<int>* result, size_t jump_limit,
+    size_t ii, const distance_matrix_storage<short>::row_ref row) const {
+
+    // Scan the LUT of system ids and add any result from the row within
+    // the neighborhood range to the results.
+    for (auto it : m_system_id_to_graph_index) {
```

Comment:
True, but range based for loops don't use the iterator directly but the type that the iterator dereferences to.